packages/launchpad/README.md

```diff
@@ -25,6 +27,14 @@ Cypress' entire back-end is powered by the `@packages/server` package. Launchpad
 
 [Here is a short tutorial](https://github.com/lmiller1990/vue-3-urql-example) building a simple app using the same technologies we are using for launchpad. There are a lot of moving pieces; understanding how everything works will help you contribute to Launchpad.
 
+## Major Version Welcome Content
+
+The content is bundled with the launchpad and at the time of writing it lives in `src/migration/MajorVersionWelcome.vue`. Shipping it as part of the app means there will never be a delay in displaying it and it will always work offline. Guidelines for the management of the content itself are documented internally in our `prod-eng-docs`, but the implementation is documented here.
+
+A constant named `MAJOR_VERSION_FOR_CONTENT` defines which major version the content is associated with for the purposes of recording user dismissal in persisted state. This needs to be bumped to match the major version that will be going out, since that value is the key that records the dismissal.
```

Comment:
```suggestion
A constant named `MAJOR_VERSION_FOR_CONTENT` defines which major version the content is associated with for the purposes of recording user dismissal in persisted state. This needs to be bumped to match the major version that will be released, since that value is the key that records the dismissal.
```

Reply:
ha, this is much more suitable